index.html

```diff
@@ -212,6 +212,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *@kudamndau* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">ldsuk00</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">@LdsUk00</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/your_user_handle" target="_blank">LdsUk00e</a>
```

Comment:
Note that you need to add the link to your profile landing page ( inside the `<a>` tag ). For example, instead of `your_user_handle` you should mention your profile URL.
```
https://www.github.com/your_user_handle
```
You should replace `your_user_handle` with **your user ID**. In your case, it should be as follows. 
```
https://github.com/lds-uk
```
### Likewise you must modify both links. 

Also kindly note that your GitHub profile user ID is `lds-uk` and not `LdsUk00e`. This URL `https://github.com/LdsUk00e` redirects to 404 Not found. 
Kindly take care of those and once check the URL before commiting.